README.md

```diff
@@ -340,3 +340,31 @@ You may also need to remove this namespace when you no longer need it.
 ```bash
 kubectl delete namespace $DESIREDNAMESPACE
 ```
+
+#### Install the Kubernetes Dashboard.
```

Comment:
We're moving towards keeping this page as simple as possible, but indeed, the dashboard could be helpful.

Could we rename this section to just 'Kubernetes Dashbord' (without the period), and something like:

> You may find it helpful to see the Kubernetes resources visually which can be achieved by installing the Kubernetes Dashboard...

which can just be a single link to the K8s page rather than duplicating and maintaining the commands here?

Reply:
Yes, I agree the page should be simpler. It does seem rather long trying to cover two topics.

Are suggesting link to Anaxes / AWS / Kops, as per:

https://github.com/Alfresco/alfresco-anaxes-shipyard/blob/master/docs/running-a-cluster.md#set-up-and-start-kops-cluster (section 5)

Or:

https://github.com/kubernetes/dashboard/wiki/Installation

However, it might good to have all info in one place rather than links to another project / page.

Reply:
Sure, no worries - although I do link it above. I suggest two options:

1.

#### Kubernetes Dashboard

You may find it helpful to see the Kubernetes resources visually which can be achieved by installing the Kubernetes Dashboard...

https://github.com/kubernetes/dashboard/wiki/Installation

2. As above but also add:

The above describes two approaches. The "recommended approach" which is more secure and the "alternative approach" which makes access easier in a local development environment. Here's a quick summary of the latter:

```bash
kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/alternative/kubernetes-dashboard.yaml
```

To access the dashboard, you can start local proxy server.

```bash
$ kubectl proxy
```

Once proxy server is started you should be able to access Dashboard from your browser at:

http://localhost:8001/api/v1/namespaces/kube-system/services/http:kubernetes-dashboard:/proxy/

If we feel that  option 2 will help developers (new to Docker Desktop) then let me know and I'll add it back in :-)

Reply:
Can you please give me contributor right to push directly to this project ?

I will then update this PR with option 1 for now (unless I hear otherwise).

Reply:
Done.

And the link can just be inline, I was just being brief in the quote.

Reply:
Done

Reply:
I'm not seeing those changes in this PR, can you update or create a new one?

Reply:
https://github.com/Alfresco/alfresco-dbp-deployment/commit/7f17d51ae3b16b39ed977bc940ad74a6c98a3160